doc/getting-started/apl-nuc.rst

```diff
@@ -47,41 +47,50 @@ complete this setup.
    and follow the `Clear Linux installation guide
    <https://clearlinux.org/documentation/clear-linux/get-started/bare-metal-install>`__
    as a starting point for installing Clear Linux onto your platform.  Follow the recommended
-   options for choosing an **Automatic** installation type, and using the platform's
+   options for choosing an **Manual** installation type, and using the platform's
    storage as the target device for installation (overwriting the existing data
    and creating three partitions on the platform's storage drive).
+   
+   High-level steps should be: 
+   
+   #.  Install Clear on a NUC Use the "Manual" option.
+   #.  Use default partition scheme for storage
+   #.  Name the host "clr-sos-guest"
+   #.  Add an administrative user "clear" with "sudoers" privilege
+   #.  Add these additional bundles "editors", "user-basic", "desktop-autostart", "network-basic"
+   #.  For network, choose “DHCP”
 
 #. After installation is complete, boot into Clear Linux, login as
-   **root**, and set a password.
+   **clear**, and set a password.
 
 #. Clear Linux is set to automatically update itself. We recommend that you disable
    this feature to have more control over when the updates happen. Use this command
    to disable the autoupdate feature:
 
    .. code-block:: none
 
-      # swupd autoupdate --disable
+      $ sudo swupd autoupdate --disable
 
    .. note::
       The Clear Linux installer will automatically check for updates and install the
       latest version available on your system. If you wish to use a specific version
       (such as 26200), you can achieve that after the installation has completed using
-      ``swupd verify --fix --picky -m 26200``
-
+      ``sudo swupd verify --fix --picky -m 26200``
+       
 #. If you have an older version of Clear Linux already installed
    on your hardware, use this command to upgrade Clear Linux
    to version 26200 (or newer):
 
    .. code-block:: none
 
-      # swupd update -m 26200     # or newer version
+      $ sudo swupd update -m 26200     # or newer version
 
-#. Use the ``swupd bundle-add`` command and add these Clear Linux bundles:
+#. Use the ``sudo swupd bundle-add`` command and add these Clear Linux bundles:
 
    .. code-block:: none
 
-      # swupd bundle-add vim sudo network-basic service-os kernel-iot-lts2018 \
-          openssh-server software-defined-cockpit
+      $ sudo swupd bundle-add service-os kernel-iot-lts2018 \
+          openssh-server
```

Comment:
Is there any specific reason why you are not adding those bundles already as part of the manual installation steps?

Reply:
OK